pkg/api/grpcServer.go

```diff
@@ -177,8 +177,8 @@ func (s *grpcServer) dispatch(stream grpc.Stream, methodName string, getState st
 				glog.Errorf("Unable to send gRPC response message: %v", err)
 			}
 
-			requestBag.Done()
-			responseBag.Done()
+			attribute.BagDone(&requestBag)
```

Comment:
The value it adds is if the requestBag is used after this point, it will be `nil` and produce a hard failure.